frontend/src/pages/cookie-policy.tsx

```diff
@@ -0,0 +1,137 @@
+import { useAuth } from '../api/auth';
+import Layout from '../components/layouts/Main';
+import LoaderOverlay from '../components/LoaderOverlay';
+import './i18n';
+import { useTranslation } from 'react-i18next';
+
+const Cookies = () => {
+  const { user } = useAuth();
+  const { t, i18n, ready } = useTranslation(['common']);
+  return (
+    <>
+      {!user ? (
+        <LoaderOverlay />
+      ) : (
+        <Layout title={`Climate Policy Radar | ${t('Cookie policy')}`}>
+          <section>
+            <div className="text-content px-4 container mb-12">
+              <h1 className="my-8">Cookie Policy</h1>
+              <p>
+                Read about what cookies are, which cookies Climate Policy Radar
+                (CPR) use on the Climate Policy Radar Website (CPR Website and
+                Ap), and how can you manage your cookies.
+              </p>
+              <h2>What are cookies?</h2>
+              <p>
+                Cookies are small files that are downloaded onto your computer
+                when you visit a website. They are used by many websites to make
+                them work, or to help them work more efficiently. They also
+                provide us with information about how you use the website.
+              </p>
+              <p>
+                Website operators are required by law - specifically the{' '}
+                <span className="italic">
+                  Privacy and Electronic Communications (EC Directive)
+                  (Amendment) Regulations 2011 (UK Regulations)
+                </span>{' '}
+                - to provide clear and comprehensive information about the use
+                of cookies, and ensure that users have consented to this use.
+              </p>
+              <h2>Which Cookies do we Use?</h2>
+              <p>
+                Various cookies are in use across the CPR Website and App, with
+                the cookies in use subject to changes as the website develops.
+                These remain in your browser's subfolder and are activated again
+                when you re-visit the website. It helps us to remember your
+                information and settings when you visit the website again,
+                resulting in a faster online experience for you.
+              </p>
+              <h3>Google Analytics</h3>
+              <p>__ga__gat__</p>
+              <p className="italic">
+                Used to monitor usage of the CPR Website and App. This
+                information allows us to assess how the Website is used.
+                Anonymous data about usage (eg number of visitors, visitors'
+                country, number of times each page is visited) is collected and
```

Comment:
I'm not sure about this one, but "visitors' country" feels wrong... maybe ignore for now or check with a better grammar checker :joy: 

```suggestion
                Anonymous data about usage (eg number of visitors, visitors'
                countries, number of times each page is visited) is collected and
```

Reply:
the more I look at this one the less confident I feel about my suggestion :joy:

Reply:
This would be something you could raise with Michal, as this is something she wrote.